src/app/icd/client/DefaultICDClientStorage.cpp

```diff
@@ -39,7 +39,11 @@ static_assert(kMaxFabricListTlvLength <= std::numeric_limits<uint16_t>::max(), "
 
 namespace chip {
 namespace app {
-CHIP_ERROR DefaultICDClientStorage::UpdateFabricList(FabricIndex fabricIndex)
+
+inline constexpr uint8_t kAppDataLength = 6;
```

Comment:
This should go in the `ICDClientStorage.h` since it is common to all implementations.